settings/CHIPIR/galil/chipir_set_velocities_again.bat

```diff
@@ -0,0 +1,15 @@
+caput IN:CHIPIR:MOT:MTR0201.VELO 0.9734124833021497
```

Comment:
This `set_velocities_again` file was my fault and is no longer required (it's just grepped down from `chipir.bat` anyway)

Reply:
will remove